geode-core/src/main/java/com/gemstone/gemfire/internal/security/IntegratedSecurityService.java

```diff
@@ -0,0 +1,164 @@
+package com.gemstone.gemfire.internal.security;
+
+import java.util.Properties;
+import java.util.concurrent.Callable;
+
+import org.apache.geode.security.ResourcePermission;
+import org.apache.geode.security.SecurityManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.shiro.subject.Subject;
+import org.apache.shiro.util.ThreadState;
+
+import com.gemstone.gemfire.internal.logging.LogService;
+import com.gemstone.gemfire.management.internal.security.ResourceOperation;
+
+public class IntegratedSecurityService implements SecurityService {
+
+  private static Logger logger = LogService.getLogger(LogService.SECURITY_LOGGER_NAME);
+
+  private static SecurityService defaultInstance = new IntegratedSecurityService();
+
```

Comment:
Can someone just call new IntegratedSecurityService()? If so, probably just create a private constructor here to prevent that.